core/server/api/v2/utils/serializers/output/settings.js

```diff
@@ -22,7 +22,13 @@ _private.settingsFilter = (settings, filter) => {
 
 module.exports = {
     browse(models, apiConfig, frame) {
-        let filteredSettings = _.values(_private.settingsFilter(models, frame.options.type));
+        let filteredSettings;
+        // If this is public, we already have the right data, we just need to add an Array wrapper
+        if (frame.options.context.public) {
```

Comment:
There is a helper fn called [isContentAPI](https://github.com/TryGhost/Ghost/blob/2.9.1/core/server/api/v2/utils/index.js#L18).